tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
+
 	private $tmpDir;
 
+	protected function getPort()
+	{
+		return $this->startPort + $this->triedPorts;
+	}
+
+	protected function getIPAddress()
+	{
+		$records = dns_get_record(php_uname('n'), DNS_A);
```

Comment:
I am not sure whether you can rely on this working. Perhaps, this might not work on Travis which is usually rather weird. What's the problem of leaving the default of 127.0.0.1 (ie, localhost)?